src/scenes/Wallet/components/Balances/index.jsx

```diff
@@ -70,6 +78,11 @@ class MyBalances extends React.PureComponent {
       <Paper className={classes.myBalancePaper}>
         <Grid container spacing={0} className={classes.myBalanceGridContainer}>
           <Typography variant="title" className={classes.myBalanceTitle}>
+            <div className={classes.importantNoteContainer}>
```

Comment:
instead of adding an extra wrapper `<div>`, why don't you just add `margin-bottom` in a class to `<ImportantNote />`?
You'll have to extend `ImportantNote` by putting `<div {...props}>` in it